src/scripts/party/PartyPokemon.ts

```diff
@@ -117,6 +118,12 @@ class PartyPokemon implements Saveable {
                 this.addOrRemoveHeldItem(this.heldItem());
             }
         });
+        this.clickAttackBonus = ko.computed((): number => {
+            // Caught + Shiny + Resistant
+            const bonus = 1 + +this.shiny + +(this.pokerus >= GameConstants.Pokerus.Resistant);
+            const heldItemMultiplier = this.heldItem && this.heldItem() instanceof HybridAttackBonusHeldItem ? (this.heldItem() as HybridAttackBonusHeldItem).clickAttackBonus : 1;
```

Comment:
```suggestion
            const heldItemMultiplier = this.heldItem() instanceof HybridAttackBonusHeldItem ? (this.heldItem() as HybridAttackBonusHeldItem).clickAttackBonus : 1;
```
First check is truish at all time due to `this.heldItem` pointing to the observable. However, it means it is not required as the second check sorts out `undefined` anyway.

Reply:
I copied it from the regular pokemon attack held item bonus check, lol. Thanks